### DISCUSSION TITLE
Install cli on windows

### DISCUSSION THREAD

**Discussion by Arctomachine** [supabase/supabase#29070] [Questions]
According to https://supabase.com/docs/guides/cli/getting-started?queryGroups=platform&platform=windows#installing-the-supabase-cli, it requires some scoop to install it. But whatever this scoop is, it doesnt seem to be present in system by default, which makes installing cli extra inconvenient.
Is there better way to install it using only native tools?

**Comment by silentworks** [comment]
There is a link to scoop on the word Scoop. You can also install the CLI using `npx` if you don't wish to install scoop. The `npx` option works across all operating systems as long as you have `npm` installed. https://supabase.com/docs/guides/cli/getting-started?queryGroups=platform&platform=npx#installing-the-supabase-cli
